src/bentoml_cli/deployment.py

```diff
@@ -83,41 +85,67 @@ def parse_runners_callback(
         # ("'a'", "'b'")
     return parsedLs
 
+
 class Mutex(click.Option):
```

Comment:
It is not really Mutex option anymore is it? I think this is RequiredAllMutexOption

Reply:
It differs a bit imo, here `[--deployment-name, --bento]` and `[--file]` are required & mutually exclusive, but other options except `--output` are forbidden too when using `--file`. but they are optional

Reply:
so if users pass in --file it should be executed immediately right?

Reply:
in terms of setting context and the other variable for create from file we can tell user with the --file workflow to use environment variable to set for that variable specifically

Reply:
`--output` and `--context` are valid options when using `--file`. I agree `context` could be stored as an environment variable, but `--output` might be better to be passed in?...